### ISSUE TITLE
Different output from GAPIT Farmcpu and rMVP Farmcpu

### ISSUE THREAD

**Issue by shameem356** [xiaolei-lab/rMVP#57]
Hello,

I was using same set of data in GAPIT Farmcpu and rMVP Farmcpu (the genotype file in not imputed ).We have got very significant result from  GAPIT Farmcpu. But rMVP Farmcpu is showing different result compared to GAPIT Farmcpu. Is it possible ?

**Comment by XiaoleiLiuBio** [comment]
Yes, this is possible if the genotype data was not fully imputed, and also the default sets of FarmCPU function-related parameters are different between GAPIT and rMVP.

**Comment by shameem356** [comment]
Hello @XiaoleiLiuBio ,
Thank you so much for the reply. Can we use genotype file in rMVP Farmcpu which is in not imputed,  ? If yes, how much reliable the answer ?  my rMVP results SNPs are  functionally very good as compared to GAPIT. So please can you clarify this case ?

**Comment by XiaoleiLiuBio** [comment]
All functions of the association tests, e.g., GLM, MLM, and FarmCPU don't accept genotype data with missing values. Please impute the genotype data by professional imputation tools before using rMVP. If only a small proportion of genotype values are missing, you can use rMVP to impute the missing genotypic values by major alleles.
